packages/flutter_tools/lib/src/isolated/resident_web_runner.dart

```
@@ -928,6 +952,18 @@ class ResidentWebRunner extends ResidentRunner {
       _registeredMethodsForService.remove(serviceName);
     }
   }
+
+  /// Checks if the given JSON response indicates no clients are available.
+  /// Returns an [OperationResult] if no clients are available, otherwise returns null.
+  OperationResult? _checkNoClientsAvailable(Map<String, dynamic>? json, Status status) {
+    if (json != null && json['noClientsAvailable'] == true) {
+      _logger.printTrace('No browser clients currently connected');
+      status.stop();
+      _logger.printStatus(kNoClientConnectedMessage);
+      return OperationResult.ok;
```

Should this be `ok`? Or are we doing this to suppress an error?

---

Yes, that’s right. That’s currently what we’re doing when attempting a hot reload without the initial DWDS connection. We do this to exit the method early before running into an error.

For reference: [resident_web_runner.dart#L492](https://github.com/flutter/flutter/blob/master/packages/flutter_tools/lib/src/isolated/resident_web_runner.dart#L492)